src/targets/services/checkAndUpdate.js

```diff
@@ -0,0 +1,35 @@
+import CozyClient from 'cozy-client'
+import { schema } from '../../helpers/doctypes'
+import log from 'cozy-logger'
+import {
+  updateContact,
+  getContactsToUpdate,
+  getLastSuccessOfService
+} from '../../helpers/updateContact'
+import { omit } from 'lodash'
+
+export const service = async () => {
+  log('info', `Executing checkAndUpdate service`)
+  const client = CozyClient.fromEnv(process.env, { schema })
+
+  try {
+    const lastSuccess = await getLastSuccessOfService(client, 'checkAndUpdate')
+    const contactsToUpdate = await getContactsToUpdate(client, lastSuccess)
+    const updatedContactsToUpload = contactsToUpdate.map(
+      // omit is to prevent updateAll error : Bad special document member: _type
```

Comment:
Can you add a link to the issue that will solve the problem, this way we can remove the hack when it's fixed.